CMakeLists.txt

```diff
@@ -631,8 +658,8 @@ endif ()
 
 # -- compiler setup ------------------------------------------------------------
 
-set(CLANG_MINIMUM_VERSION 11.0)
-set(APPLE_CLANG_MINIMUM_VERSION 12.0.5)
+set(CLANG_MINIMUM_VERSION 13.0)
+set(APPLE_CLANG_MINIMUM_VERSION 14.0)
```

Comment:
Note to the reviewer: This was just out of sync with our actual requirements, so I updated it.